plugins/modules/aci_l3out_extepg.py

```diff
@@ -49,6 +49,36 @@
     type: str
     choices: [ AF11, AF12, AF13, AF21, AF22, AF23, AF31, AF32, AF33, AF41, AF42, AF43, CS0, CS1, CS2, CS3, CS4, CS5, CS6, CS7, EF, VA, unspecified ]
     aliases: [ target ]
+  contract_exception_tag:
+    description:
+    - The tag for contract exception.
+    type: str
+  qos_class:
+    description:
+    - The priority class identifier.
+    type: str
+    choices: [ level1, level2, level3, level4, level5, level6, unspecified ]
+    default: level3
+  intra_ext_epg_isolation:
+    description:
+    - The preferred policy control.
+    type: str
+    choices: [ enforced, unenforced ]
+  route_control_profiles:
```

Comment:
do we need a dict here, or should we provide import_route_control_profile and export_route_control_profile?

Reply:
think this would make the overal logic more simple

Reply:
Already had a discussion and we decided to go this route